reg/udral/README.md

```diff
@@ -236,6 +236,27 @@ It is not necessary to include the leading and trailing name component separator
 Registers that are not known to the auto-configuration authority are to be silently ignored,
 which permits addition of implementation-defined ports without breaking compatibility with standard services.
 
+
+#### Type signature register
+
+In many use cases it is desirable to have the ability to statically check the compatibility between sender and 
+receiver of a message. This capability could be used both during automatic configuration and by manual configurator tools.
+The design allows this by adding an extra register to every service pub/sub group that contains a string _message type signature_.
+
+    uavcan.sub.main_drive.setpoint.type_sig
+
+Example signature would look like `(u12{u8}a5[f16]=i8)` , and be interpreted as
+
+ - `()` delimit sealed type
+ - `{}` delimit extensible type
+ - `b`, `u`, `i`, `f`, `v` followed by size in bits encode primitive types
+ - `a` and `A` followed by length in items and brackets encode variable and fixed size arrays
+ - `=` marks byte alignment point
+
+Sample code that generates type string from DSDL and compares two strings for compatibility can be seen at this 
+(temporary) [location](https://gitlab.com/vadimz1/dsdl-sig-gen/-/tree/main).
```

Comment:
Embedding links into text like this makes the text unintelligible when copy-pasted.

```suggestion
(temporary) location: <https://gitlab.com/vadimz1/dsdl-sig-gen/-/tree/main>.
```

Reply:
> Embedding links into text like this makes the text unintelligible when copy-pasted.

Hmm, not sure what you are saying here - the  []() format was exactly the same as you used above with github link.

Reply:
My point is that the text should be meaningful when copy-pasted without the embedded link. Ex: "can be seen at this temporary location" is meaningless because the referenced location is not specified, "can be seen at Vadim's GitLab" is fine because the location is specified even if the embedded hyperlink is lost.